tests/console/opensuse_repos.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: openSUSE-repos
+# Summary: Basic checks for openSUSE-repos.
+
+# - Install openSUSE-repos
+# - Innstall openSUSE-repos-NVIDIA on aarch64 and x86_64
+# - Import gpg keys and refresh repositories
+# - Flavors of openSUSE-repos packages correspond with flavor of openSUSE-release
+# Maintainer: Lubos Kocmman <lubos.kocman@suse.com>
```

Comment:
Feel free to add me as maintainer as well if you want to share the burden :wink:

Reply:
will happily do!